lib/fluri.dart

```diff
@@ -271,11 +271,11 @@ class FluriMixin {
       // Add the param value(s) while eliminating duplicates.
       // Throw an ArgumentError if any value is invalid.
       if (value is String && !newQueryParameters.containsValue(value)) {
-        newQueryParameters[key].add(value);
+        newQueryParameters[key]!.add(value);
       } else if (value is Iterable<String>) {
         for (var v in value) {
-          if (!newQueryParameters[key].contains(v)) {
-            newQueryParameters[key].add(v);
+          if (!newQueryParameters[key]!.contains(v)) {
+            newQueryParameters[key]!.add(v);
```

Comment:
These 3 `!`s are also ok because of https://github.com/Workiva/fluri/pull/85/files#diff-ee6083449fc9ed8e925a843b6dc1013c5e28e335204b4c0bd8de3b7a81265963R259-R261